CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+### Fixed
```

Comment:
Please revert this. The file is auto-generated

Reply:
ohhh sorry I didn`t now this